src/mon/PGMap.cc

```diff
@@ -955,16 +954,17 @@ void PGMapDigest::dump_object_stat_sum(
         tbl << "N/A";
       else
         tbl << stringify(si_u_t(pool->quota_max_objects));
-
       if (pool->quota_max_bytes == 0)
         tbl << "N/A";
       else
         tbl << stringify(byte_u_t(pool->quota_max_bytes));
-
-      tbl << stringify(si_u_t(sum.num_objects_dirty))
-	  << stringify(byte_u_t(statfs.data_compressed_allocated))
-	  << stringify(byte_u_t(statfs.data_compressed_original))
-	  ;
+      if (pool->is_tier()) {
+        tbl << stringify(si_u_t(sum.num_objects_dirty))
+	      << stringify(byte_u_t(statfs.data_compressed_allocated))
+	      << stringify(byte_u_t(statfs.data_compressed_original));
+      }
+      else
+        tbl << "N/A";
```

Comment:
```suggestion
      } else {
        tbl << "N/A";
      }
```

also. this breaks the tests

```
[ RUN      ] pgmap.dump_object_stat_sum_0
../src/test/mon/PGMap.cc:103: Failure
Expected equality of these values:
  stringify(si_u_t(sum.num_objects_dirty))
    Which is: "2"
  tbl.get(0, col++)
    Which is: "N/A"
[  FAILED  ] pgmap.dump_object_stat_sum_0 (0 ms)
[ RUN      ] pgmap.dump_object_stat_sum_1
../src/test/mon/PGMap.cc:137: Failure
Expected equality of these values:
  stringify(si_u_t(0))
    Which is: "0"
  tbl.get(0, col++)
    Which is: "N/A"
[  FAILED  ] pgmap.dump_object_stat_sum_1 (0 ms)
```

Reply:
i don't think the test failures were addressed.